client/src/components/DisplayTable/DisplayTable.tsx

```diff
@@ -533,19 +552,11 @@ export class _DisplayTable extends React.Component<
                   colSpan={_.size(visible_col_keys)}
                 >
                   <div
-                    style={{ display: "flex", justifyContent: "space-between" }}
+                    style={{
+                      display: "flex",
+                      justifyContent: "space-between",
+                    }}
                   >
-                    <div>
-                      {enable_pagination && (
-                        <SelectPageSize
-                          selected={page_size}
-                          on_select={this.change_page_size}
-                          page_size_increment={page_size_increment}
-                          num_items={_.size(sorted_filtered_data)}
-                          num_options_max={page_size_num_options_max}
-                        />
-                      )}
-                    </div>
                     <div className={"display-table-container__utils"}>
                       <button
```

Comment:
This button should actually be the first tabbable element, so it should go before `selectPageUtil`, so all of this div can be cleaned up and the button can move up